src/main/kotlin/no/nav/familie/ef/sak/behandlingsflyt/steg/BehandlingSteg.kt

```diff
@@ -59,6 +59,11 @@ enum class StegType(
         tillattFor = BehandlerRolle.BESLUTTER,
         gyldigIKombinasjonMedStatus = listOf(BehandlingStatus.FATTER_VEDTAK)
     ),
+    ANGRE_SEND_TIL_BESLUTTER(
+        rekkefølge = 4,
+        tillattFor = BehandlerRolle.SAKSBEHANDLER,
+        gyldigIKombinasjonMedStatus = listOf(BehandlingStatus.FATTER_VEDTAK)
+    ),
```

Comment:
Er lagt til to steg som har rekkefølge 4, er dette greit?

Alle tester blir grønne og rekkefølgen brukes ikke direkte

Reply:
Tror det går fint, ja. Det viktige er kombinasjonen og tillatt for

Reply:
Trenger vi ett eget steg for denne? Det er vel strengt tatt ikke et steg, man er på `BESLUTTE_VEDTAK` og når man avbryter så kommer man vel tilbake til `SEND_TIL_BESLUTTER`?